lib/features/popup-menu/PopupMenu.js

```diff
@@ -122,6 +122,7 @@ PopupMenu.prototype._render = function() {
         entries=${ entriesArray }
         headerEntries=${ headerEntriesArray }
         scale=${ scale }
+        emitEvent=${ this._emit.bind(this) }
```

Comment:
Please expose standard `onOpen` and `onClose` handlers.

Reply:
Is there a reason why we emit closed throught the PREACT component? We should be fine with emitting it directly from the popup menu.

Reply:
Whatever you decide to go for, let's adhere to a common components convention: Props that are passed into a component are called `on{NAME}`. If you go the generic _emit any event_ route, call the thing `onEvent` or `onLifeCycleEvent`. I'd advocate for a specific handler though.